devtools/create_engine/create_engine.cpp

```diff
@@ -0,0 +1,163 @@
+// create_engine.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <ctype.h>
+#include <sys/stat.h>
+#include <Windows.h>
+
+// Specified engine name with different cases
+#define MAX_LINE_LENGTH 256
+char engineUppercase[MAX_LINE_LENGTH];
+char engineCamelcase[MAX_LINE_LENGTH];
+char engineLowercase[MAX_LINE_LENGTH];
+
+// List of files to be copied to create engine
+static const char *const FILENAMES[] = {
+	"configure.engine", "console.cpp", "console.h",
+	"credits.pl", "detection.cpp", "detection.h",
+	"detection_tables.h", "metaengine.cpp",
+	"metaengine.h", "module.mk", "xyzzy.cpp", "xyzzy.h",
```

Comment:
For consistency, it would be better to put 3 entries per line

Reply:
Done